From 4c2a6684be57b7c6400199b2cb124019e5a04f64 Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Fri, 13 Jul 2018 20:25:48 +0000
Subject: [PATCH] Conditionally pack tests as tools based on source-build

https://github.com/dotnet/roslyn-tools/issues/291 is required to remove this patch.

(cherry picked from commit 258b703931d01be047030d3c5160316a35c331ac)
---
 src/Tests/Directory.Build.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Tests/Directory.Build.targets b/src/Tests/Directory.Build.targets
index dc4a5cc2..6d8e62ec 100644
--- a/src/Tests/Directory.Build.targets
+++ b/src/Tests/Directory.Build.targets
@@ -5,7 +5,7 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(IsTestProject)' == 'true' AND '$(OutputType)' == 'Exe'">
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true' AND '$(OutputType)' == 'Exe' AND '$(DotNetBuildFromSource)' != 'true'">
     <IsPackable>true</IsPackable>
     <PackAsTool>true</PackAsTool>
     <ToolCommandName>$(PackageId)</ToolCommandName>
-- 
2.17.1.windows.2

